empress_wrapper_examples.py

```diff
@@ -0,0 +1,74 @@
+# TODO: To be converted to automated tests
```

Comment:
This is only for showing what it will probably look like. Haven't implemented or tested the implementation.